src/WalkingModule/src/Module.cpp

```diff
@@ -1274,6 +1303,8 @@ bool WalkingModule::setPlannerInput(double x, double y)
 
     m_newTrajectoryRequired = true;
 
+    double treshold = 0.01;
+
```

Comment:
Is this needed?

Reply:
Fixed in a9a2591e7e1e2a6a93e5ff5a08cfdcb51d9b3003